packages/contentstack-import/src/utils/import-config-handler.ts

```diff
@@ -98,6 +95,16 @@ const setupConfig = async (importCmdFlags: any): Promise<ImportConfig> => {
   config.replaceExisting = importCmdFlags['replace-existing'];
   config.skipExisting = importCmdFlags['skip-existing'];
 
+  if (importCmdFlags['exclude-global-modules']) {
+    config['exclude-global-modules'] = importCmdFlags['exclude-global-modules'];
+  }
+
+  if (importCmdFlags['mapper-dir'] && importCmdFlags['exclude-global-modules']) {
```

Comment:
This whole condition can be removed

Reply:
Second condition mandate the mapper dir flag, make it optional if provided use it

Reply:
Remvoed